tests/functional_tests/triggers_test.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2023 Agnostiq Inc.
+#
+# This file is part of Covalent.
+#
+# Licensed under the GNU Affero General Public License 3.0 (the "License").
+# A copy of the License may be obtained with this software package or at
+#
+#      https://www.gnu.org/licenses/agpl-3.0.en.html
+#
+# Use of this file is prohibited except in compliance with the License. Any
+# modifications or derivative works of this file must retain this copyright
+# notice, and modified files must contain a notice indicating that they have
+# been altered from the originals.
+#
+# Covalent is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the License for more details.
+#
+# Relief from the License may be granted by purchasing a commercial license.
+
+"""
+Testing whether Triggers functionality works as expected
+"""
+
+import time
+from pathlib import Path
+
+import covalent as ct
+from covalent.triggers import DirTrigger, TimeTrigger
+
+
+def test_dir_trigger():
```

Comment:
Add one-liner doc string

Reply:
@FyzHsn this is the file containing the functional tests for the triggers feature.

Reply:
ah I see 😅